worldengine/simulations/precipitation.py

```diff
@@ -40,20 +45,46 @@ def _calculate(seed, width, height):
         octaves = 6
         freq = 64.0 * octaves
 
+        n_scale = 1024 / float(height) #This is a variable I am adding. It exists
+                                       #so that worlds sharing a common seed but
+                                       #different sizes will have similar patterns
```

Comment:
What I mean is that if I generate a world with seed 1 at 1024 x 1024 it will have the same general temperature and precipitation maps as a world generated at 512 x 512, simply at a larger size. It is a step towards allowing people to generate 'low resolution' maps quickly until they find something they like and then taking the time to generate a higher resolution map without totally changing the results.

There's a lot of other steps that will need to be done before this fully works. Right now the land mass itself will completely change, but this is a first step.